utils/util_test.go

```diff
@@ -27,3 +28,37 @@ func TestBitsBytesConvert(t *testing.T) {
 		}
 	}
 }
+
+func TestIntToBytes(t *testing.T) {
+	numInt := 42
+	b := IntToBytes(numInt)
+	if int(binary.LittleEndian.Uint32(b)) != numInt {
+		t.Fatal("Conversion to bytes looks wrong!")
+	}
+	numInt = -42
+	b = IntToBytes(numInt)
+	if int32(binary.LittleEndian.Uint32(b)) != int32(numInt) {
```

Comment:
Wait, I changed another line; but here it's needed or not? `int32(binary.LittleEndian.Uint32(b)) != numInt` won't work because the type of `numInt` is an `int` and the type of `int32(binary.LittleEndian.Uint32(b))` is an `int32`